app/docs/components/kbd/kbd.mdx

```diff
@@ -0,0 +1,177 @@
+
+The Kbd (Keyboard) component can be used to indicate a textual user input from the keyboard inside other elements such as in text, tables, cards, and more.
+
+Use the semantic `<Kbd>` keyboard input tag to use the default monospace font which is best suited for representing input keys.
+
+To start using the `<Kbd>` component you need to import it from `flowbite-react`:
+
+```jsx
+'use client';
+
+import { Kbd } from 'flowbite-react';
+```
+
+## Table of Contents
+
+## Default KBD
+
+Here’s a list of KBD components that you can use inside any other element.
+
+<CodePreview importFlowbiteReact="Kbd" title="Default Kbd">
+  <Kbd>Shift</Kbd>
+  <Kbd>Ctrl</Kbd>
+  <Kbd>Tab</Kbd>
+  <Kbd>Caps Lock</Kbd>
+  <Kbd>Esc</Kbd>
+  <Kbd>Spacebar</Kbd>
+  <Kbd>Enter</Kbd>
+</CodePreview>
+
+## KBD inside text
+
+Use this example by nesting an inline KBD component inside a paragraph.
+
+<CodePreview importFlowbiteReact="Kbd" title="Inside Text Kbd">
+  Please press <Kbd>Ctrl</Kbd> + <Kbd>Shift</Kbd> + <Kbd>R</Kbd> to re-render an MDN page.
+</CodePreview>
+
+## KBD inside table
+
+The KBD component can also be used inside table components to denote what type of key can be pressed for certain descriptions.
+
+<CodePreview
+  importFlowbiteReact="Kbd, Table"
+  importExternal="import { MdKeyboardArrowLeft, MdKeyboardArrowRight, MdKeyboardArrowDown, MdKeyboardArrowUp  } from 'react-icons/md';"
+  title="Table Kbd"
+>
+  <Table>
+    <Table.Head>
+      <Table.HeadCell>Key</Table.HeadCell>
+      <Table.HeadCell>Description</Table.HeadCell>
+    </Table.Head>
+    <Table.Body className="divide-y">
+      <Table.Row className="bg-white dark:border-gray-700 dark:bg-gray-800">
+        <Table.Cell className="whitespace-nowrap font-medium text-gray-900 dark:text-white">
+          <Kbd>Shift</Kbd> <span className="pr-1">or</span> <Kbd>Tab</Kbd>
+        </Table.Cell>
+        <Table.Cell>Navigate to interactive elements</Table.Cell>
+      </Table.Row>
+      <Table.Row className="bg-white dark:border-gray-700 dark:bg-gray-800">
+        <Table.Cell className="whitespace-nowrap font-medium text-gray-900 dark:text-white">
+          <Kbd>Enter</Kbd> <span className="pr-1">or</span> <Kbd>Spacebar</Kbd>
+        </Table.Cell>
+        <Table.Cell>Ensure elements with ARIA role="button" can be activated with both key commands.</Table.Cell>
+      </Table.Row>
+      <Table.Row className="bg-white dark:border-gray-700 dark:bg-gray-800">
+        <Table.Cell className="whitespace-nowrap font-medium text-gray-900 dark:text-white">
+          <Kbd icon={MdKeyboardArrowUp}></Kbd>
+          <Kbd icon={MdKeyboardArrowDown}></Kbd>
+          <span className="pr-1">or</span>
+          <Kbd icon={MdKeyboardArrowLeft}></Kbd>
+          <Kbd icon={MdKeyboardArrowRight}></Kbd>
+        </Table.Cell>
+        <Table.Cell>Choose and activate previous/next tab.</Table.Cell>
+      </Table.Row>
+    </Table.Body>
+  </Table>
+</CodePreview>
+
+## Arrow keys
```

Comment:
I think the arrows are different from the flowbite documentation, aren't they? Is there any particular reason?

Reply:
🤔 I think that the standard flowbite library is now using the flowbite icons, but we don't support it yet. So, just ignore it and leave it as it is for now.